src/sentry/integrations/jira/client.py

```diff
@@ -16,59 +16,6 @@
 CUSTOMFIELD_PREFIX = "customfield_"
 
 
-class JiraCloud:
-    """
-    Contains the jira-cloud specifics that a JiraClient needs
-    in order to communicate with jira
-    """
-
-    def __init__(self, shared_secret):
-        self.shared_secret = shared_secret
-
-    @property
-    def cache_prefix(self):
-        return "sentry-jira-2:"
-
-    def request_hook(self, method, path, data, params, **kwargs):
-        """
-        Used by Jira Client to apply the jira-cloud authentication
-        """
-        # handle params that are already part of the path
-        url_params = dict(parse_qs(urlsplit(path).query))
-        url_params.update(params or {})
-        path = path.split("?")[0]
-
-        jwt_payload = {
-            "iss": JIRA_KEY,
-            "iat": datetime.datetime.utcnow(),
-            "exp": datetime.datetime.utcnow() + datetime.timedelta(seconds=5 * 60),
-            "qsh": get_query_hash(path, method.upper(), url_params),
-        }
-        encoded_jwt = jwt.encode(jwt_payload, self.shared_secret)
-        params = dict(jwt=encoded_jwt, **(url_params or {}))
-        request_spec = kwargs.copy()
-        request_spec.update(dict(method=method, path=path, data=data, params=params))
-        return request_spec
-
-    def user_id_field(self):
-        """
-        Jira-Cloud requires GDPR compliant API usage so we have to use accountId
-        """
-        return "accountId"
-
-    def user_query_param(self):
-        """
-        Jira-Cloud requires GDPR compliant API usage so we have to use query
-        """
-        return "query"
-
-    def user_id_get_param(self):
-        """
-        Jira-Cloud requires GDPR compliant API usage so we have to use accountId
-        """
-        return "accountId"
-
-
 class JiraApiClient(ApiClient):
```

Comment:
super hyper optional nit: If we're calling the client for Jira Server `JiraServerClient`, it'd be nice if this one was called `JiraCloudClient` instead of `JiraApiClient`

Reply:
Or just JiraClient, since that's the naming style used in `integration.py` as well. 👍